MovieDirectory/PresentationLayer/Program.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using MovieDirectory;
+using DataMovie;
+
+
+namespace MovieDirectory
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            var movies = new MovieVault();
+            ChooseAnAction(movies);
+        }
+        public static void ChooseAnAction(MovieVault movies)
+        {
+            while (true)
+            {
+                Console.WriteLine("Choose an action:\n-create\n-read\n-update\n-delete\n-exit");
+                var action = Console.ReadLine().ToLower();
+
+                switch (action)
+                {
+                    case "create":
+                        Create(movies);
+                        break;
+                    case "read":
+                        Read(movies);
+                        break;
+                    case "update":
+                        Update(movies);
+                        break;
+                    case "delete":
+                        Delete(movies);
+                        break;
+                    case "exit":
+                        return;
+                    default:
+                        Console.WriteLine("Incorrect input. Try again.");
+                        break;
+                }
+            }
+        }
+        public static void Create(MovieVault movie)
+        {
+            Console.WriteLine("Enter movie name:");
+            string name = Console.ReadLine();
+
+            Console.WriteLine("Enter the id of the film:");
+            if (!int.TryParse(Console.ReadLine(), out int id) || movie.movies.ContainsKey(id))
```

Comment:
`movie.movies.ContainsKey(id)` breaks encapsulation. May be you can create a separate method in your Vault to make this check?